src/_guides/language/analysis-options.md

```diff
@@ -100,15 +101,17 @@ and `my_other_other_package`, and file #2 to analyze the code in
 `my_package`.
 
 
-## Enabling Dart 2 semantics
+## Enabling Dart 2 semantics (SDK \<2.0.0-dev.68.0) {#enabling-dart-2-semantics}
 
 {% comment %}
 update-for-dart-2
 {% endcomment %}
 
-The analyzer in the Dart SDK doesn't yet use the
-[latest Dart type system][sound-dart], by default.
-The simplest way to enable Dart 2 semantics is to put
+The analyzer, in Dart SDK releases 2.0.0-dev.68.0 and later,
+uses the [latest Dart type system][sound-dart] by default.
+
+The simplest way to enable Dart 2 semantics in earlier
+dev releases of the Dart 2 SDK is to put
```

Comment:
"dev" isn't necessary here.

Reply:
Below, please change "Performing additional type checks" from a 3rd-level heading to a 2nd-level heading.